### ISSUE TITLE
Not working when not using Chrome

### ISSUE THREAD

**Issue by RobSchoenaker** [sipsorcery/sipsorcery-media#7]
WebRTC with Chrome on localhost works fine. However, when I use Firefox or Edge to connect through WebRTC, for example to the WebRTCReceiver, I get the following error:

`07/07/2020 13:35:40|Error|<>c__DisplayClass71_0.<receiveRequest>b__0|Authentication failed, see inner exception.`

I do not know how to fix this. Any hints?

**Comment by sipsorcery** [comment]
The error is due to the browser not accepting the web socket SSL certificate. The reason it's not accepted is because it's a self-signed localhost certificate. Unfortunately each browser has its own way of trusting a self-signed certificate. Take a look at the comments in this [file](https://github.com/sipsorcery/sipsorcery-media/blob/master/examples/WebRtcDaemon/WebRTCDaemon.cs). I added those notes last year when I went through the same process on Chrome, FireFox and Edge (Chromium).
